projects/emissary-ingress/emissary/helm/patches/0001-helm-changes.patch

```diff
@@ -4501,15 +4717,327 @@ index 2d06c8127..f1e043136 100644
            imagePullPolicy: {{ .Values.image.pullPolicy }}
            ports:
              {{- range .Values.service.ports }}
+@@ -190,7 +190,7 @@ spec:
+             {{- end }}
+             - name: AMBASSADOR_NAMESPACE
+               {{- if .Values.namespace.name }}
+-              value: {{ .Values.namespace.name | quote }}
++              value: {{ .Values.namespace.name |.Release.Namespace | default .Values.defaultNamespace | quote }}
```

Comment:
If we already have `Values.namespace.name`, why do we need `.Values.defaultNamespace`?

Reply:
We should replace Values.namespace.name with .Values.defaultNamespace to match with what package controller expects.

Reply:
Values.namespace.name is only for the commercial product, and used an env variable.